core-web/libs/portlets/edit-ema/portlet/src/lib/components/dot-ema-dialog/dot-ema-dialog.component.spec.ts

```diff
@@ -325,5 +325,16 @@ describe('DotEmaDialogComponent', () => {
 
             expect(resetSpy).toHaveBeenCalled();
         });
+
+        it("should trigger openDialogOnURL in the store when it's a URL", () => {
+            const openDialogOnURLSpy = jest.spyOn(storeSpy, 'openDialogOnURL');
+
+            component.openDialogOnUrl('https://demo.dotcms.com/jsp.jsp', 'test');
+
+            expect(openDialogOnURLSpy).toHaveBeenCalledWith({
+                title: 'test',
+                url: 'https://demo.dotcms.com/jsp.jsp'
+            });
+        });
```

Comment:
Let's never do this, when we need to call a method in a component, let's use a service, when I did the refactor of the toolbar this was the only pain point.

Reply:
Noted, will consider this in the future.